pillar/consul/mitx.sls

```diff
@@ -19,22 +19,4 @@ consul:
           check:
             tcp: '{{ mysql_endpoint }}'
             interval: 10s
-        {% for cache_config in env_data.get('backends', {}).get('elasticache', []) %}
```

Comment:
Why not remove previous lines starting from line13 where the if statement is?

Reply:
Because we currently rely on the DNS name being `mysql.service.consul` and if I remove that then it will break that resolution.